From 2bc13405df0d287fbe9afe8edf1fc31f930678df Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Mon, 15 May 2023 01:55:05 +0000
Subject: [PATCH] set some runtime script interpreters to /bin/sh

nothing in these scripts make use of bash features, and they are
actually posix sh, so use /bin/sh instead.
---
 data/carla                     | 2 +-
 data/carla-bridge-lv2-modgui   | 2 +-
 data/carla-control             | 2 +-
 data/carla-database            | 2 +-
 data/carla-jack-multi          | 2 +-
 data/carla-jack-patchbayplugin | 2 +-
 data/carla-jack-single         | 2 +-
 data/carla-libdir              | 2 +-
 data/carla-osc-gui             | 2 +-
 data/carla-patchbay            | 2 +-
 data/carla-rack                | 2 +-
 data/carla-settings            | 2 +-
 data/copy-zynaddsubfx          | 2 +-
 13 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/data/carla b/data/carla
index c20d4e41b..4b6def81e 100755
--- a/data/carla
+++ b/data/carla
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-bridge-lv2-modgui b/data/carla-bridge-lv2-modgui
index dba0c9661..ebe709ed1 100644
--- a/data/carla-bridge-lv2-modgui
+++ b/data/carla-bridge-lv2-modgui
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-control b/data/carla-control
index c9dd5f73b..3338fee16 100755
--- a/data/carla-control
+++ b/data/carla-control
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-database b/data/carla-database
index debb1893e..0aebdd130 100755
--- a/data/carla-database
+++ b/data/carla-database
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-jack-multi b/data/carla-jack-multi
index b1b0e61c2..5caef0cce 100755
--- a/data/carla-jack-multi
+++ b/data/carla-jack-multi
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-jack-patchbayplugin b/data/carla-jack-patchbayplugin
index 6c39c91cf..d47b19390 100755
--- a/data/carla-jack-patchbayplugin
+++ b/data/carla-jack-patchbayplugin
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 if [ -f "$(dirname ${0})/carla-utils.pc" ]; then
     cd "$(dirname ${0})/.."
diff --git a/data/carla-jack-single b/data/carla-jack-single
index 3de70aa9f..64ffacb97 100755
--- a/data/carla-jack-single
+++ b/data/carla-jack-single
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-libdir b/data/carla-libdir
index 48341008b..192f90813 100755
--- a/data/carla-libdir
+++ b/data/carla-libdir
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # small script that only prints carla's libdir.
 # can be used to detect if carla is installed, and where to find its libcarla_standalone2.so file
 
diff --git a/data/carla-osc-gui b/data/carla-osc-gui
index 6e02c5a20..5a1f2871c 100755
--- a/data/carla-osc-gui
+++ b/data/carla-osc-gui
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
diff --git a/data/carla-patchbay b/data/carla-patchbay
index 76aba4d42..83bafb08f 100755
--- a/data/carla-patchbay
+++ b/data/carla-patchbay
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-rack b/data/carla-rack
index 3fae861f0..7462511e5 100755
--- a/data/carla-rack
+++ b/data/carla-rack
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/carla-settings b/data/carla-settings
index a53bb77b3..da6c8b817 100755
--- a/data/carla-settings
+++ b/data/carla-settings
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 PYTHON=$(which python3 2>/dev/null)
 
diff --git a/data/copy-zynaddsubfx b/data/copy-zynaddsubfx
index f38c968d7..53aa1ef51 100755
--- a/data/copy-zynaddsubfx
+++ b/data/copy-zynaddsubfx
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
-- 
2.40.1

